src/Core/Silk.NET.BuildTools/Bind/ClassWriter.cs

```diff
@@ -443,6 +443,7 @@ public static void WriteMixedModeClasses(this Project project, Profile profile,
                         sw.WriteLine("}");
                         sw.WriteLine();
                         sw.Flush();
+                        sw.Dispose();
```

Comment:
Why not use the `using` syntax here instead of manual disposal?

Reply:
Because I didn't have the time to evaluate whether it would be safe to do so from a scope POV. I essentially just did a Ctrl+F for `Flush` and whacked a `Dispose` after each call.

Reply:
ah, makes sense